app/build.gradle

```diff
@@ -43,10 +43,16 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.3.1'
     implementation 'com.google.android.material:material:1.4.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.1'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation 'androidx.navigation:navigation-fragment-ktx:2.5.0'
+    implementation 'androidx.navigation:navigation-ui-ktx:2.5.0'
+    implementation 'androidx.security:security-crypto-ktx:1.1.0-alpha03'
     testImplementation 'junit:junit:4.+'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
     implementation "androidx.security:security-crypto:1.1.0-alpha01"
     implementation "com.google.code.gson:gson:2.9.0"
 
+    //circule Img
```

Comment:
we don't need circule image handeler 
we can handel it using image load lib like picasso or glid 
or hndel from XML  using ShapeableImageView witch it a material componant

Reply:
thanks for the tip, will use one of them ISA